mevblocker/web
# Cookie policy

# Summary

Adds Cookie policy page based on [this file
](https://docs.google.com/document/d/1_G9YNuLxLhosWBAmNmHxgwFjeGlUCH6iDpl_IuPppYo/edit)

# To test
1. scroll down to the footer
2. click on Cookie policy link
3. you should see the cookie policy page
